main.cpp

```diff
@@ -52,106 +52,115 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: lion
+//  action 1: the lion eats prey
+lion.eatPrey();
+//  action 2: the lion sleeps in a cave
+lion.sleepInCave();
+//  action 3: the lion feeds her children
+lion.feedChildren();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: dolphin
+//  action 1: the dolphin jumps out of water
+lion.jumpOutWater();
+//  action 2: the dolphin fights a shark
+dolphin.fightShark();
+//  action 3: the dolphin becomes sentient
+dolphin.discoverReality();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Brooklyn
+//  action 1: Brooklyn snows
+brooklyn.heavySnow();
+//  action 2: Brooklyn opens a new park
+brooklyn.createPark();
+//  action 3: Brooklyn just hosted a marathon
+brooklyn.hostMarathon();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Michael Jordan
+//  action 1: Michael Jordan releases a new shoe
+mj.releaseNewShoe();
+//  action 2: MJ shoots three point attempt
+mj.shootThreePointer();
+//  action 3: MJ answers post game questions
+mj.answerReporter();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Spongebob
+//  action 1:  Spongebob makes a krabby patty
+  spongebob.cookKrabbyPatty();
+//  action 2: Spongebob feeds Gary
+  spongebob.feedGary();
+//  action 3: Spongebob tries to catch jellyfish
+  spongebob.jellyfishCatchAttempt();

```

Comment:
The plain english description doesn't match the pseudo-code.
They should align 1:1 or nearly 1:1
- [ ] revise